catalog/core/catalog-core-api/src/main/java/ddf/catalog/operation/CreateRequest.java

```diff
@@ -26,9 +27,9 @@
  */
 public interface CreateRequest extends Request {
   /**
-   * Gets a {@link List} of {@link Metacard} to be stored.
+   * Gets a {@link Set} of {@link Metacard} to be stored.
    *
-   * @return the {@link List} of {@link Metacard}
+   * @return the {@link Set} of {@link Metacard}
    */
-  public List<Metacard> getMetacards();
+   Set<Metacard> getMetacards();
```

Comment:
❗️ Please note that this interface is whitelisted and cannot be changed without architectural input and a plan of action for deprecation/evolution of the contract.

(https://github.com/codice/ddf/blob/master/distribution/docs/src/main/resources/content/_whitelists/catalog-whitelist.adoc)